eventfinder/node_modules/.cache/babel-loader/431e0daf5c3ea1ed7870b955492751c2.json

```diff
@@ -0,0 +1 @@
+{"ast":null,"code":"var _jsxFileName = \"/Users/dylanneil/ReactProject/eventfinder/src/App.js\";\nimport React from 'react';\nimport './App.css';\nimport MuiThemeProvider from \"@material-ui/core/styles/MuiThemeProvider\";\nimport Button from \"@material-ui/core/Button\";\nimport theme from \"./theme\";\nimport { makeStyles } from '@material-ui/core/styles';\nimport Eventcard from './eventcard.js';\n\nfunction App() {\n  return /*#__PURE__*/React.createElement(\"div\", {\n    className: \"App\",\n    __self: this,\n    __source: {\n      fileName: _jsxFileName,\n      lineNumber: 12,\n      columnNumber: 5\n    }\n  }, /*#__PURE__*/React.createElement(\"header\", {\n    className: \"App-body\",\n    __self: this,\n    __source: {\n      fileName: _jsxFileName,\n      lineNumber: 13,\n      columnNumber: 7\n    }\n  }, /*#__PURE__*/React.createElement(Eventcard, {\n    __self: this,\n    __source: {\n      fileName: _jsxFileName,\n      lineNumber: 16,\n      columnNumber: 9\n    }\n  })));\n}\n\nexport default App;","map":{"version":3,"sources":["/Users/dylanneil/ReactProject/eventfinder/src/App.js"],"names":["React","MuiThemeProvider","Button","theme","makeStyles","Eventcard","App"],"mappings":";AAAA,OAAOA,KAAP,MAAkB,OAAlB;AACA,OAAO,WAAP;AACA,OAAOC,gBAAP,MAA6B,2CAA7B;AACA,OAAOC,MAAP,MAAmB,0BAAnB;AACA,OAAOC,KAAP,MAAkB,SAAlB;AACA,SAASC,UAAT,QAA2B,0BAA3B;AACA,OAAOC,SAAP,MAAsB,gBAAtB;;AAGA,SAASC,GAAT,GAAe;AACb,sBACE;AAAK,IAAA,SAAS,EAAC,KAAf;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA,kBACE;AAAQ,IAAA,SAAS,EAAC,UAAlB;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA,kBAGE,oBAAC,SAAD;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA;AAAA,IAHF,CADF,CADF;AAUD;;AAED,eAAeA,GAAf","sourcesContent":["import React from 'react';\nimport './App.css';\nimport MuiThemeProvider from \"@material-ui/core/styles/MuiThemeProvider\";\nimport Button from \"@material-ui/core/Button\";\nimport theme from \"./theme\";\nimport { makeStyles } from '@material-ui/core/styles';\nimport Eventcard from './eventcard.js';\n\n\nfunction App() {\n  return (\n    <div className=\"App\">\n      <header className=\"App-body\">\n\n\n        <Eventcard />\n\n      </header>\n    </div>\n  );\n}\n\nexport default App;\n"]},"metadata":{},"sourceType":"module"}
```

Comment:
Need to remove node_modules from the commit history

Reply:
/node_modules has been added to .gitignore.